packages/text-annotator/src/TextAnnotatorOptions.ts

```diff
@@ -17,6 +17,8 @@ export interface TextAnnotatorOptions<I extends TextAnnotation = TextAnnotation,
 
   presence?: PresencePainterOptions;
 
+  selectionMode?: 'shortest' | 'all';
```

Comment:
Should we add documentation about meaning of those properties right in the code?
```suggestion
  /**
   * Defines the behavior when the user selects an existing annotation
   * that overlaps other annotations via the pointer event.
   * - `shortest` (default): selects the shortest annotation at the click position.
   * - `all`: will select the whole stack of annotations at the click position,
   *          in other words, all annotations that intersect the position.
   */
  selectionMode?: 'shortest' | 'all';
```